### DISCUSSION TITLE
Action Items RSEAA24

### DISCUSSION THREAD

**Discussion by rowlandm** [RSEAA/RSEAA.github.io#146] [Discussion topics for #RSEAA24]
List of action items that we might want to consider in the RSE community and possibly for next year

**Comment by manodeep** [comment]
Set of few (~3-5) short tech-talks? Say with a frequency of twice per year

**Reply by jyoti-bhogal** [reply]
In order for the community realize that there is continuity in the tech talk/hands-on workshop, the frequency of the same should be monthly one talk/workshop rather than having the sessions once on a 3 monthly basis.

**Comment by manodeep** [comment]
Meeting with funders

**Comment by manodeep** [comment]
Collect RSE Impact stories (perhaps as an globally collaborative action)

**Comment by klapaukh** [comment]
Membership survey looking at job responsibilities, salaries, etc.

**Comment by junranlei** [comment]
Quarterly discussion topic or theme (can vote here) to keep the discussion going;
A resource and toolkits list people can enter input in.

**Comment by manodeep** [comment]
Add a speed networking session for RSEs

**Comment by manodeep** [comment]
Create a map of RSEs, potentially creating collaborations between RSEs across domains, seniority, tech stacks and interests

**Comment by jyoti-bhogal** [comment]
Skill Development hands-on workshop

**Comment by jyoti-bhogal** [comment]
For those working remotely, we can have co-working sessions once a week - people are free to chat or go into a breakout room if they want to work silently

**Comment by rowlandm** [comment]
Concrete career path definition for technical and non-techinical RSE

**Reply by rowlandm** [reply]
What does the community want when they mean career path.

**Reply by rowlandm** [reply]
There could be more than one path.

- Management
- Technical Lead
- Professor

**Reply by manodeep** [reply]
Junior RSE, RSE, Senior RSE, Lead RSE, Principal RSE?

**Reply by rowlandm** [reply]
Technical Lead might be Level C type thing

**Reply by rowlandm** [reply]
Software architectures  and enterprise artchitects

**Reply by klapaukh** [reply]
We could also look at properties of jobs that people want. Do we want, for example:
 - A pathway for pay rises
 - Changing of job titles over time
 - Job security
 - etc?

**Comment by rowlandm** [comment]
Focus on non-technical connections to help you remain employed as an Research Software

**Reply by rowlandm** [reply]
When we first did this in 2019 at eResearch, I was surprised no-one was that interested in tech.

**Reply by rowlandm** [reply]
I went through ~150 member requests for the RSE-AUNZ google group and put together a spreadsheet that has a very rough view of what members wanted to get out of the community.

Generic Comment | count | Personal comment
-- | -- | --
Looking for jobs | 17 | We are sharing jobs as part of the regular digest we send before events
Connecting with other RSEs | 16 | We do provide opportunities for synchronous but not asynchronous communication. RSEAA24 has a Slack workspace now that could potentially help .
Stay up to date | 13 | We do provide updates as part of the regular digest we send before events. Unsure if these updates are appropriate to the community though  (ie. tech news, policy development, skills ?).
Aligns with my job | 12 |  That shows they identify as RSEs.
Looking for RSEs to hire | 3 | How can we connect hiring managers with RSEs both proactively and reactively?
Recently started a job that this aligns with | 3 | That shows they identify as RSEs.
Share knowledge and experience | 3 | Don't always do this well proactively.
Leadership in the sector / Contribute to the Community | 2 | How can we imrpove recruitment and retention of volunteers who want to contribute back?

**Reply by rowlandm** [reply]
Making it easier to move to over to other domains. What happens when my contract ends?

**Reply by rowlandm** [reply]
eg. Cheat sheet for concrete terms - make it easier for RSEs to get up to speed with researchers. Also helpful for research infrastructure.

**Comment by jyoti-bhogal** [comment]
RSE Talent Register

**Reply by rowlandm** [reply]
Doing better education and outreach around building teams for complex projects.

**Reply by rowlandm** [reply]
eg. recruitment and position descriptions.

**Comment by rowlandm** [comment]
Sharing suggestions on how to help improve specific things while working with students at scale.

**Reply by rowlandm** [reply]
this is what I was suggesting for you Anusha!

**Comment by rowlandm** [comment]
partner with the Carpentries

**Reply by rowlandm** [reply]
Education resource. Ties into education and outreach.

**Reply by rowlandm** [reply]
RSE College of Experts, or perhaps even consultants to advise RSE managers/Directors.

Maybe event a Complex project management Research organisation.

**Comment by junranlei** [comment]
Center of Excellence Concept Model - provides leadership, best practices, research, support, training of trainers and skill training...

**Comment by manodeep** [comment]
What skills should University graduates (undergrads, PhDs) and postdocs have to ensure they can line up an industry job? And can RSEs help provide such training + hands-on projects?
